uniffi_bindgen/src/bindings/kotlin/gen_kotlin.rs

```diff
@@ -16,18 +16,23 @@ use crate::MergeWith;
 #[derive(Debug, Default, Clone, Serialize, Deserialize)]
 pub struct Config {
     package_name: Option<String>,
+    cdylib_name: Option<String>,
 }
 
 impl Config {
-    fn default_package_name() -> String {
-        "uniffi".into()
-    }
```

Comment:
I didn't see the point in adding a `default_cdylib_name()` method and maybe more in future, so I just inlined the defaults into the methods below.